packages/openneuro-components/src/search-page/search-page.scss

```diff
@@ -377,6 +377,17 @@
   }
 }
 
+.third-party {
+  margin: 10px 0;
+}
+
+.third-party .on-accordion-wrapper .keyword-accordion {
+  .accordion-title {
+    position: absolute;
+    left: 300px;
+  }
+}
+
```

Comment:
One suggestion is this can be better if the CSS is specific to the component. It's a bit messy here since we're reusing some global styles that haven't been moved to their own component.

```typescript
import styled from "@emotion/styled"

const NeurobagelSearchDiv = styled.div`
  padding-bottom: 10px;
  .on-accordion-wrapper .keyword-accordion {
    .accordion-title {
        position: absolute;
        top: -20px;
        left: 280px;
    }
  }
`
```

Reply:
Thanks for the suggestion @nellh! We refactored things into a `NeurobagelSearch` component, applied the emotion-styling, and added Neurobagel to the link for clarity.